src/TwigComponent/doc/index.rst

```diff
@@ -7,7 +7,7 @@ making it easier to render and re-use small template "units" - like an
 
 Every component consists of (1) a class::
 
-    // src/Twig/Components/Alert.php
+    // src/Twig/Component/Alert.php
```

Comment:
```suggestion
    // src/Twig/Component/Alert.php
```

Something we will discuss in 3.0 .. but i totally agree.

To be fair, i even think we should use `src/Component`

Reply:
i do have "devland codebase" component (relatively to symfony component)
having them at root level with "twig component" seems wrong to me